File: javascript/bubblesort.js
function bubbleSort(lista) {

    const index_maximo = lista.length-1;

    for (i = index_maximo; i>0; i--){
        for(j = 0; j<i; j++){
            if(lista[j] > lista[j+1]){
                console.log(lista);
                [lista[j], lista[j+1]] = [lista[j+1], lista[j]];
            }
        }
    }
    return lista
}

let numeros = [7, 5, 2, 0, 4, 9, 3]

let result = bubbleSort(numeros)
console.log("Resultado")
console.log(result)

File: ordenamento/bubble_sort.py
def BubbleSort(lista):
    for index_final in range(len(lista), 0, -1):
        for index in range(0, index_final-1):
            if lista[index] > lista[index+1]:
                print(lista)
                lista[index+1], lista[index] = lista[index], lista[index+1]
    return lista

numeros = [4,7,2,8,5,11,0,-1,7]
print(BubbleSort(numeros))